docs/_posts/SKocer/2023-05-11-icd10cm_cause_claim_mapper_en.md

```diff
@@ -0,0 +1,119 @@
+[Copy S3 URI](s3://auxdata.johnsnowlabs.com/clinical/models/icd10cm_cause_claim_mapper_en_4.4.0_3.0_1683819210044.zip){:.button.button-orange.button-orange-trans.button-icon.button-copy-s3}
+
+## How to use
+
+
+
+<div class="tabs-box" markdown="1">
+{% include programmingLanguageSelectScalaPythonNLU.html %}
+```python
+document_assembler = DocumentAssembler()\
+      .setInputCol("text")\
+      .setOutputCol("document")
+
+chunk_assembler = Doc2Chunk()\
+      .setInputCols("document")\
+      .setOutputCol("icd_chunk")
+
+chunkerMapper = ChunkMapperModel.pretrained("icd10cm_cause_claim_mapper", "en", "clinical/models")\
+      .setInputCols(["icd_chunk"])\
+      .setOutputCol("mappings")\
+      .setRels(["icd10cm_cause", "icd10cm_claim_analysis_code"])
+
+pipeline = Pipeline().setStages([document_assembler,
+                                 chunk_assembler,
+                                 chunkerMapper])  
+
+model = pipeline.fit(spark.createDataFrame([['']]).toDF('text')) 
+
+lp = LightPipeline(model)
+
+res = lp.fullAnnotate(["D69.51", "G43.83", "Z83.51"])
+```
+```scala
+val document_assembler = new DocumentAssembler()
+       .setInputCol("text")
+       .setOutputCol("document")
+
+val chunk_assembler = new Doc2Chunk()
+      .setInputCols("document")
+      .setOutputCol("icd_chunk")
+
+val chunkerMapper = ChunkMapperModel
+       .pretrained("icd10cm_cause_claim_mapper", "en", "clinical/models")
+       .setInputCols(Array("icd_chunk"))
+       .setOutputCol("mappings")
+       .setRels(Array("icd10cm_cause", "icd10cm_claim_analysis_code")) 
+
+val mapper_pipeline = new Pipeline().setStages(Array(document_assembler, chunk_assembler, chunkerMapper))
+
+val data = Seq(Array("D69.51", "G43.83", "Z83.51")).toDS.toDF("text")
+
+val result = pipeline.fit(data).transform(data) 
+```
+</div>
+
+## Results
+
+```bash
++------+------------------------------------+---------------------------+
+|icd10 |mappings                            |relation                   |
++------+------------------------------------+---------------------------+
+|D69.51|Unintentional injuries              |icd10cm_cause              |
+|D69.51|Adverse effects of medical treatment|icd10cm_cause              |
+|D69.51|D69.51                              |icd10cm_claim_analysis_code|
+|D69.51|D69.51                              |icd10cm_claim_analysis_code|
+|G43.83|Headache disorders                  |icd10cm_cause              |
+|G43.83|Migraine                            |icd10cm_cause              |
+|G43.83|Tension-type headache               |icd10cm_cause              |
+|G43.83|G43.83                              |icd10cm_claim_analysis_code|
+|G43.83|G43.83                              |icd10cm_claim_analysis_code|
+|G43.83|G43.83                              |icd10cm_claim_analysis_code|
+|Z83.51|NONE                                |null                       |
```

Comment:
change here. And also try to find some examples that icd10cm codes are not the same icd10cm claim analysis code.